libs/blocks/marquee/marquee.js

```diff
@@ -133,16 +121,6 @@ export default function init(el) {
   text.classList.add('text');
   const media = foreground.querySelector(':scope > div:not([class])');
 
-  if (media) {
-    media.classList.add('media');
-
-    if (media.querySelector('a[href$=".mp4"]')) {
-      decorateVideo(media);
-    } else {
-      decorateImage(media);
-    }
-  }
-
   const firstDivInForeground = foreground.querySelector(':scope > div');
```

Comment:
If you are removing media class and above logic, you may want to check the css as well those using `media` class.

Reply:
I realized that the decorateImage function is still needed as well.  Added back in.